test/server/cards/02_SHD/units/GreySquadronYWing.spec.ts

```diff
@@ -0,0 +1,31 @@
+describe('Grey squadron Y-Wing', function() {
+    integration(function(contextRef) {
+        describe('Grey squadron Y-Wing\'s Ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        spaceArena: ['grey-squadron-ywing'],
+                        groundArena: ['wampa'],
+                    },
+                    player2: {
+                        groundArena: ['battlefield-marine'],
+                        spaceArena: ['cartel-spacer'],
+                        base: { card: 'echo-base', damage: 0 }
```

Comment:
Can just omit the base definition completely and use `context.p2Base` for targeting purposes